pkg/flag/misconf_flags.go

```diff
@@ -55,6 +55,12 @@ var (
 		Default:    []string{},
 		Usage:      "specify paths to override the Terraform tfvars files",
 	}
+	CfParamsFlag = Flag{
+		Name:       "cf-params",
```

Comment:
CF scanner also has an option to pass parameters as key value (key1=value1,key2=value2). If we want to add support for this option in the future, will the choice of flag name cause problems?

Reply:
I think it's cumbersome to pass key=value pairs via a command line option so I think it's fine for now. We don't plan to support such a case as it is error prone on user input / shell.